discovery-provider/src/api/v1/playlists.py

```diff
@@ -328,48 +335,42 @@ class FullTrendingPlaylists(Resource):
         }
     )
 
-    def get_cache_key(self):
-        request_items = to_dict(request.args)
-        request_items.pop('limit', None)
-        request_items.pop('offset', None)
-        key = extract_key(request.path, request_items.items())
-        return key
-
     @record_metrics
     @full_ns.marshal_with(full_trending_playlists_response)
     def get(self):
         """Get trending playlists"""
-        # Parse args
         args = full_trending_parser.parse_args()
-        offset, limit = format_offset(args), format_limit(args, TRENDING_LIMIT)
-        current_user_id, time = args.get("user_id"), args.get("time", "week")
-        time = "week" if time not in ["week", "month", "year"] else time
+        strategy = trending_selector.get_strategy(TrendingType.PLAYLISTS)
+        playlists = trending_playlists(request, args, strategy)
+        return success_response(playlists)
 
-        # If we have a user_id, we call into `get_trending_playlist`
-        # which fetches the cached unpopulated tracks and then
-        # populates metadata. Otherwise, just
-        # retrieve the last cached value.
-        #
-        # If current_user_id,
-        # apply limit + offset inside the cached calculation.
-        # Otherwise, apply it here.
-        if current_user_id:
-            args = {
-                'time': time,
-                'with_tracks': True,
-                'limit': limit,
-                'offset': offset
-            }
-            decoded = decode_string_id(current_user_id)
-            args["current_user_id"] = decoded
-            playlists = get_trending_playlists(args)
-        else:
-            args = {
-                'time': time,
-                'with_tracks': True,
-            }
-            key = self.get_cache_key()
-            playlists = use_redis_cache(key, TRENDING_TTL_SEC, lambda: get_trending_playlists(args))
-            playlists = playlists[offset: limit + offset]
+@full_ns.route("/trending/<string:version>")
```

Comment:
Does this actually need to be a distinct endpoint? Couldn't the regular trending_playlists endpoint optionally take a version string, and fall back to default if none is provided?